src/test/java/io/strimzi/kafka/bridge/http/HttpBridgeTest.java

```diff
@@ -59,7 +60,7 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 @ExtendWith(VertxExtension.class)
-@SuppressWarnings({"checkstyle:JavaNCSS"})
+@SuppressWarnings({"checkstyle:JavaNCSS", "checkstyle:ClassDataAbstractionCoupling"})
```

Comment:
adding the `HealthChecker` was the need for this new suppress warning rule?

Reply:
Yes.